packages/rspack/rspack.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
   entry: {
     main: './src/main.ts',
   },
+  devServer: {
```

Comment:
rspack will inject hmr code in development mode by default(which will be turned off in 0.5), so disable it